### ISSUE TITLE
How to create a useful issue

### ISSUE THREAD

**Issue by luckysmg** [luckysmg/flutter_swipe_action_cell#13]
### 1. If you come across some bug
you can try lastest version of this package.

### 2.  If The bugs are found in lastest version,you should:
   
- provide demo code for me to reproduce bug quickly.The code can be run directly
- provide a description for me .It can be text,or GIF or video to show me where the bug is.

### If you have some enhancement advice
just write in issue directly.😀

**Comment by nick-named** [comment]
Im not sure how it works exactly but the delete animation could be better for the Customize Shape example. I'm using the original animation for delete (instead of having it blank) and it looks a bit odd. 

Not a big thing, it just looks off.

cheers

**Comment by luckysmg** [comment]
yes....If you customize your shape ....You 'd better not use this animation....Because the background will not follow behind the button @nick-named
